docs/aws_s3_compat.md

```diff
@@ -176,6 +176,7 @@ See also `GetObject` and other method parameters.
 | 🟢 | HeadBucket           |           |
 | 🟢 | ListBuckets          |           |
 | 🔵 | PutPublicAccessBlock |           |
+| 🔴 | GetPublicAccessBlock |           |
```

Comment:
Likely it's blue and depending on the method above.

Reply:
Sure